e2e/_suites/fleet/stand-alone.go

```diff
@@ -177,35 +145,52 @@ func (sats *StandAloneTestSuite) startAgent(image string, composeFilename string
 		return err
 	}
 
-	sats.Image = image
-	sats.Hostname = hostname
-	sats.Cleanup = true
+	fts.Image = image
+	fts.Hostname = hostname
 
-	err = sats.installTestTools(containerName)
+	err = fts.installTestTools(containerName)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (sats *StandAloneTestSuite) getContainerLogs() error {
-	serviceManager := compose.NewServiceManager()
+func (fts *FleetTestSuite) theIntegrationIsAddedToThePolicy(packageName string) error {
+	return theIntegrationIsOperatedInThePolicy(fts.kibanaClient, fts.FleetServerPolicy, packageName, "added")
+}
+
+func (fts *FleetTestSuite) thePolicyShowsTheDatasourceAdded(packageName string) error {
+	fts.StandAlone = true
```

Comment:
Is this line required? I can see two different steps calling `thePolicyShowsTheDatasourceAdded`, one for standalone and one for "regular" fleet enrollments. See `fleet.go:L164` and `fleet.go:L181`. We could reuse the existing Gherkin step, or if we need to distinguish both modes, we could create another function calling the above one, but setting `.Standalone = true` before it.

Reply:
right, that was a leftover - no needed. fixed